DefInjected/StatDef/Stats_Pawns_WorkMedical.xml

```diff
@@ -9,16 +9,16 @@
   <!-- EN: medical surgery success chance -->
   <MedicalSurgerySuccessChance.label>szansa na powodzenie operacji</MedicalSurgerySuccessChance.label>
   <!-- EN: The base chance that a character will succeed when performing a medical operation. The actual success chance is also affected by factors like facilities, room cleanliness, medicine used, the difficulty of the surgery, and inspirations. No matter how high the success chance, there is always a small chance of failure on any operation. -->
-  <MedicalSurgerySuccessChance.description>Prawdopodobieństwo, z jakim operacja się powiedzie, gdy postać przystąpi do jej przeprowadzenia. Porażka oznacza zmarnowanie czasu i zasobów, a nawet katastrofalne skutki dla pacjenta.</MedicalSurgerySuccessChance.description>
+  <MedicalSurgerySuccessChance.description>Podstawowa szansa na to, że ta postać odniesie sukces podczas wykonywania zabiegu operacyjnego. Powodzenie jest także zależne od czynników, takich jak wyposażenie, czystość pokoju, zastosowany lek, trudność operacji i inspiracji. Ale bez względu na to, jak wysoka jest szansa udanego przeprowadzenia zabiegu, zawsze istnieje ryzyko niepowodzenia przy każdej operacji.</MedicalSurgerySuccessChance.description>
   
   <!-- EN: medical tend quality -->
   <MedicalTendQuality.label>jakość leczenia</MedicalTendQuality.label>
   <!-- EN: The base quality of tending given when tending wounds and illnesses. The actual tend quality will also be affected by factors like medicine used, facilities, room cleanliness, luck, and so on. -->
-  <MedicalTendQuality.description>Jakość leczenia pacjentów jako lekarz.</MedicalTendQuality.description>
+  <MedicalTendQuality.description>Podstawowa jakość otrzymanej opieki przy leczeniu ran i chorób. Jest także zależna od takich czynników, jak rodzaj stosowanego leku, wyposażenie, czystość pokoju, szczęście itp. </MedicalTendQuality.description>
```

Comment:
```suggestion
  <MedicalTendQuality.description>Wskaźnik jakości otrzymanej opieki przy leczeniu ran i chorób. Jest także zależny od takich czynników jak: rodzaj stosowanego leku, wyposażenie, czystość pokoju, szczęście itp. </MedicalTendQuality.description>
```

Reply:
Wszędzie przy "the base" tłumaczyłem na podstawowa więc niech będzie tu też do kompletu ;d